src/main/java/util/LocalDateTimePrintFormatter.java

```diff
@@ -0,0 +1,37 @@
+package util;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.TextStyle;
+import java.util.Locale;
+
+public class LocalDateTimePrintFormatter {
+
+    public static final DateTimeFormatter dateTimeFormatterForHourMin = DateTimeFormatter.ofPattern("HH:mm");
```

Comment:
피드백 반영하여 멤버 변수로 분리하여 사용하였습니다!!

Reply:
멤버 변수가 아닌 상수라고 불러주시면 됩니다! 😃 
상수는 이름을 대문자와 언더바(_) 로 네이밍하는것이 컨벤션입니다!

아래와 같이 이름도 간결하게 표현해도 좋을 것 같아요~

> 줄여 쓰지 않는다(축약 금지).

이런 규칙도 있으니 min 보다는 minute 로 사용해보죠
(min은 많이 사용하는 용어긴 하지만요... ㅎㅎ)

```suggestion
    public static final DateTimeFormatter HOUR_MINUTE = DateTimeFormatter.ofPattern("HH:mm");
```